docs/doxygen/header.html.in

```diff
@@ -70,7 +70,9 @@ $extrastylesheet
 <!--END TITLEAREA-->
 <div id="secondnav">
     <div id="versionsSelector" class="nav-placeholder"></div>
-    <div id="download-link" class="nav-placeholder"></div>
+    <!-- A proper url for download link is generated by JS -->
+    <div id="download-link" class="ovino-btn inactive"><a href="#" target="_blank">Download Docs</a></div>
     <div id="install-link" class="ovino-btn"><a href="https://software.intel.com/content/www/us/en/develop/tools/openvino-toolkit.html" target="_blank">Install OpenVINO™</a></div>
+    <div id="devcloud-link" class="ovino-btn"><a href="https://software.intel.com/content/www/us/en/develop/tools/openvino-toolkit.html" target="_blank">Try virtually on Intel&reg; DevCloud</a></div>
```

Comment:
Please change the link to https://devcloud.intel.com/edge/advanced/sample_applications/